examples/controlnet/README.md

```diff
@@ -0,0 +1,254 @@
+# Controlnet training example
+
+[Adding Conditional Control to Text-to-Image Diffusion Models](https://arxiv.org/abs/2302.05543) by Lvmin Zhang and Maneesh Agrawala.
+
+Using the pretrained models we can provide control images (for example, a depth map) to control Stable Diffusion text-to-image generation so that it follows the structure of the depth image and fills in the details. `train_controlnet.py` script shows how to train a controlnet from scratch with simple example data.
```

Comment:
```suggestion
Using the pretrained models, we can provide control images (for example, a depth map) to control Stable Diffusion text-to-image generation so that it follows the structure of the control image and steers the generated image accordingly. `train_controlnet.py` script shows how to train a ControlNet from scratch with a simple dataset.
```

Reply:
I just removed the paragraph. We have other docs that explain what controlnet is